app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,11 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
+    @user = current_user
+    @posts = current_user.last_three_posts
+  end
```

Comment:
- [ ] We need to get all the posts for a user here and display them on the post index page when a user clicks on the `See all posts` link.

```suggestion
  def index
     @user = User.find(params[:user_id])
    @posts = @user.posts.all
  end
```